src/containers/location/LocationsSagas.js

```diff
@@ -282,8 +284,8 @@ function* searchLocationsWorker(action :SequenceAction) :Generator<any, any, any
       searchInputs: searchInputs.set('selectedOption', latLonObj)
     }));
 
-    const latitude :string = isImmutable(latLonObj) ? latLonObj.get('lat') : latLonObj['lat'];
-    const longitude :string = isImmutable(latLonObj) ? latLonObj.get('lon') : latLonObj['lon'];
+    const latitude :string = isImmutable(latLonObj) ? latLonObj.get('lat') : latLonObj.lat;
+    const longitude :string = isImmutable(latLonObj) ? latLonObj.get('lon') : latLonObj.lon;
```

Comment:
@UnsungHero97's earlier comment about using immutable's functional helpers is still a great suggestion here to avoid the verbose ternary.

```
import { get } from 'immutable';

const latitude :string = get(latLonObj, 'lat');
...
```